### ISSUE TITLE
🐛 Can't run anatomical data only

### ISSUE THREAD

**Issue by XinhuiLi** [FCP-INDI/C-PAC#1575]
### Describe the bug

I couldn't run C-PAC when only anatomical data was present in the data directory. The data was organized as below:
```
└── sub-1001211
    └── ses-1
        └── anat
            └── sub-1001211_ses-1_run-1_T1w.nii.gz
```

I got the error message using default pipeline:
```
Traceback (most recent call last):
  File "/code/run.py", line 702, in <module>
    test_config = 1 if args.analysis_level == "test_config" else 0
  File "/code/CPAC/pipeline/cpac_runner.py", line 549, in run
    p_name, plugin, plugin_args, test_config)
  File "/code/CPAC/pipeline/cpac_pipeline.py", line 396, in run_workflow
    subject_id, sub_dict, c, p_name, num_ants_cores
  File "/code/CPAC/pipeline/cpac_pipeline.py", line 1016, in build_workflow
    wf, rpool = initiate_rpool(wf, cfg, sub_dict)
  File "/code/CPAC/pipeline/engine.py", line 1839, in initiate_rpool
    part_id, ses_id)
  File "/code/CPAC/pipeline/engine.py", line 1408, in ingress_raw_func_data
    func_paths_dct = data_paths['func']
KeyError: 'func'
```

### To reproduce

1. Put only anatomical data in BIDS directory
2. Run C-PAC default pipeline

### Expected behavior

C-PAC should be able to process anatomical data without errors. 

### Acceptance criteria

- [x] Process anatomical data without errors

### C-PAC version

1.8.1

### Container platform

Docker

**Comment by huguozhen** [comment]
Hello, I have the same problem. Has it been officially confirmed as a bug? How long will it take to fix? Or no plan yet?

**Comment by XinhuiLi** [comment]
> Hello, I have the same problem. Has it been officially confirmed as a bug? How long will it take to fix? Or no plan yet?

Hello! One temporary solution I found was to create a `func` folder and create a symbolic link of a fake func image (it can be a symbolic link from any func image) within this folder. So my folder structure looks like: 
```
sub-1001211
└── ses-1
    ├── anat
    │   └── sub-1001211_ses-1_run-1_T1w.nii.gz
    └── func
        └── sub-1001211_ses-1_task-rest_run-1_bold.nii.gz
```
Then I ran C-PAC with the flag `--preconfig anat-only`. The docker run command is 
```
docker run \
-v /my_data_path:/bids_dataset \
-v /my_output_path:/outputs \
-v /tmp:/scratch \
fcpindi/c-pac:develop /bids_dataset /outputs participant \
--preconfig anat-only \
--save_working_dir
```
It helped me to temporarily solve the problem. The C-PAC team will know better about the timeline to fundamentally solve it. @sgiavasis @shnizzedy @amygutierrez @tergeorge

**Comment by shnizzedy** [comment]
Resolved in #1647

**Comment by shnizzedy** [comment]
> Hello, I have the same problem. Has it been officially confirmed as a bug? How long will it take to fix? Or no plan yet?

@huguozhen ― this issue is resolved in [v1.8.3](https://github.com/FCP-INDI/C-PAC/releases/tag/v1.8.3)
